flink-table-store-hive/flink-table-store-hive-catalog/pom.xml

```diff
@@ -476,7 +476,6 @@ under the License.
                             <goal>shade</goal>
                         </goals>
                         <configuration>
-                            <minimizeJar>true</minimizeJar>
```

Comment:
Does this need? Can we add jar what we need?

Reply:
@JingsongLi, the `minimizeJar` should be false. If the `minimizeJar` is true, the hive-shims-0.23 couldn't be shaded.